CHANGELOG.md

```diff
@@ -2,6 +2,44 @@
+- chore(deps): bump winston from 3.8.1 to 3.8.2 ([#1109](https://github.com/paritytech/substrate-api-sidecar/pull/1109)) ([1c8cfc2](https://github.com/paritytech/substrate-api-sidecar/commit/1c8cfc2bdf29606db0281cbb8343afd871831b3c))
+- chore(deps): bump lru-cache from 7.13.1 to 7.14.0 ([#1107](https://github.com/paritytech/substrate-api-sidecar/pull/1107)) ([5cbc2d5](https://github.com/paritytech/substrate-api-sidecar/commit/5cbc2d51b7d0dba1d1e40f247903f406f5aee6eb))
+- chore(deps-dev): bump @substrate/dev from 0.6.4 to 0.6.5 ([#1106](https://github.com/paritytech/substrate-api-sidecar/pull/1106)) ([5a20fdf](https://github.com/paritytech/substrate-api-sidecar/commit/5a20fdfd57473a935f57e407028c5f75360bd250))
+- chore(deps): bump express and @types/express ([#1105](https://github.com/paritytech/substrate-api-sidecar/pull/1105)) ([7187a15](https://github.com/paritytech/substrate-api-sidecar/commit/7187a1568144cb3756bb72b032a8a559acb3840c))
+- chore(deps): bump path-parse from 1.0.6 to 1.0.7 ([#1120](https://github.com/paritytech/substrate-api-sidecar/pull/1120)) ([a0e69dc](https://github.com/paritytech/substrate-api-sidecar/commit/a0e69dcdad99c576f9ff7640d918df0620f9509b))
+- chore(deps): bump tar from 6.1.0 to 6.1.11 ([#1119](https://github.com/paritytech/substrate-api-sidecar/pull/1119)) ([c6ca7b1](https://github.com/paritytech/substrate-api-sidecar/commit/c6ca7b11401fb26c14341b20d471442c757061a5))
+- chore(deps): update polkadot-js deps ([#1124](https://github.com/paritytech/substrate-api-sidecar/pull/1124)) ([8cd3084](https://github.com/paritytech/substrate-api-sidecar/commit/8cd3084e79b06103122b9ea3ef46be54a935722f))
+
+### CI
+
+- ci: add dependabot for yarn ([#1096](https://github.com/paritytech/substrate-api-sidecar/pull/1096) ([24f4f26] (https://github.com/paritytech/substrate-api-sidecar/commit/24f4f26617baebdf355f84bf1f9be3070d1f4143))
+- ci: remove allow from dependabot yml ([#1101](https://github.com/paritytech/substrate-api-sidecar/pull/1101)) ([41ded6c](https://github.com/paritytech/substrate-api-sidecar/commit/41ded6cf1143594f61f45c6f8ca09606497a4b5f))
+- ci: increase bench threshold ([#1102](https://github.com/paritytech/substrate-api-sidecar/pull/1102)) ([d0defe0](https://github.com/paritytech/substrate-api-sidecar/commit/d0defe00b58cc4b13f15d01b2c45046b226ea1b0))
+- ci: allow only security updates ([#1116](https://github.com/paritytech/substrate-api-sidecar/pull/1116)) ([6f8c334](https://github.com/paritytech/substrate-api-sidecar/commit/6f8c33449ee0911078f8dc5a6e6b0a452d1ddd1a))
+
+### Bug Fixes
+- fix: adjust historyDepth to a consts ([#1115](https://github.com/paritytech/substrate-api-sidecar/pull/1115)) ([bc3c2a5](https://github.com/paritytech/substrate-api-sidecar/commit/bc3c2a5bd66232c9c40cc849b1ad03031a71c7e3))
+
+## Compatibility
+
+Tested against:
+- Polkadot v9300
+- Kusama v9300
+- Westend v9300
```

Comment:
would be good to test `westend 9310` or is it expected that it will fail?

Reply:
good catch! the version tested against was actually 9310 for westend using wss://westend.api.onfinality.io/public-ws not 9300. I was looking at Parity Polkadot v0.9.30 on polkadotjs' interface and mixed it up.